labelbox/schema/model_run.py

```diff
@@ -565,6 +569,75 @@ def export_v2(
+        """
+        Sends data rows from a model run to a project for annotation.
+
+        Args:
+            destination_project_id: The ID of the project to send the data rows to.
+            task_queue_id: The ID of the task queue to send the data rows to.  If not specified, the data rows will be
+                sent to the Done workflow state.
+            batch_name: The name of the batch to create. If more than one batch is created, additional batches will be
+                named with a monotonically increasing numerical suffix, starting at "_1".
+            data_rows: The data rows to send to the project.
+            params: Additional parameters for this operation. See SendToAnnotateFromModelParams for details.
+
+        Returns: The created task for this operation.
+
+        """
+
+        mutation_str = """mutation SendToAnnotateFromMeaPyApi($input: SendToAnnotateFromMeaInput!) {
+                            sendToAnnotateFromMea(input: $input) {
+                              taskId
+                            }
+                          }
+        """
+
+        destination_task_queue = self.client.build_destination_task_queue_input(
+            task_queue_id)
+        data_rows_query = self.client.build_catalog_query(data_rows)
+
+        model_run_ontology_mapping = params.get("model_run_ontology_mapping",
+                                                None)
+        predictions_input = self.client.build_predictions_input(
+            model_run_ontology_mapping, self.uid)
+
+        batch_priority = params.get("batch_priority", 5)
+        exclude_data_rows_in_project = params.get(
+            "exclude_data_rows_in_project", False)
+        override_existing_annotations_rule = params.get(
+            "override_existing_annotations_rule",
+            ConflictResolutionStrategy.KeepExisting)
+        res = self.client.execute(
+            mutation_str, {
+                "input": {
+                    "destinationProjectId":
+                        destination_project_id,
+                    "batchInput": {
+                        "batchName": batch_name,
+                        "batchPriority": batch_priority
+                    },
+                    "destinationTaskQueue":
+                        destination_task_queue,
+                    "excludeDataRowsInProject":
+                        exclude_data_rows_in_project,
+                    "annotationsInput":
+                        None,
+                    "predictionsInput":
+                        predictions_input,
+                    "conflictLabelsResolutionStrategy":
+                        override_existing_annotations_rule,
+                    "searchQuery": [data_rows_query],
+                    "sourceModelRunId":
+                        self.uid
+                }
+            })['sendToAnnotateFromMea']
+
+        return Entity.Task.get_task(self.client, res['taskId'])
```

Comment:
Do Task.result and Task.errors work, or any tweaks needed there to surface results/errors?

Reply:
`result` and `result_url` are not well supported with bulk-ops tasks at the moment. I will add support for surfacing errors though